extra_tests/snippets/builtin_exceptions.py

```diff
@@ -161,37 +162,45 @@ class SubError(MyError):
 assert BaseException.__new__.__qualname__ == 'BaseException.__new__'
 assert BaseException.__init__.__qualname__ == 'BaseException.__init__'
 assert BaseException().__dict__ == {}
-assert BaseException.__doc__
```

Comment:
why are these lines deleted?

Reply:
Because I now test all exceptions to have `__doc__` here: https://github.com/RustPython/RustPython/pull/2974/files#diff-9948afa9cc8e4a175f769771bdb1930355f76b99f3dface36d968c35561b7d1dR202-R206